platform/ui/src/components/SidePanel/SidePanel.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Button, Icon } from '@ohif/ui';
+
+const borderSize = 4;
+const expandedWidth = 320;
+const collapsedWidth = 60;
```

Comment:
Maybe these values can be a non-required prop? And then you set the default values in the `defaultProps`.

Reply:
I think we can leave it to the pass 2.